examples/Dashboard/src/SyncStatusWidget.tsx

```diff
@@ -14,20 +13,17 @@ import { mapSyncError, useSync } from '@openmsupply-client/system';
 
 const FormattedSyncDate = ({ date }: { date: Date | null }) => {
   const t = useTranslation('common');
-  const { localisedDistanceToNow, relativeDateTime } = useFormatDateTime();
```

Comment:
For some reason had issues with having relativeDate time and localiseDistanceToNow (they were both rendering in the same place, so removed